client/blocks/inline-help/popover.jsx

```diff
@@ -96,6 +122,88 @@ class InlineHelpPopover extends Component {
 		);
 	};
 
+	getTask = () => {
+		const task = find(
+			this.props.tasks,
+			( { id, completed } ) => ! completed && ! get( this.props.taskStatuses, [ id, 'completed' ] )
+		);
+		return task;
+	};
+
+	shouldShowChecklist = () => {
```

Comment:
The same logic found in another file. Should consider creating a shared module.

Reply:
Good call, I was wondering about that; where would be a good place to put that function?